docs/cli.md

```diff
@@ -368,6 +368,220 @@ Output will look like this, where `python` will be green as it is the package th
  xz                5.2.6       h166bdaf_0          408.6 KiB  conda  xz-5.2.6-h166bdaf_0.tar.bz2
 ```
 
+## `tree`
+
+Display the project's packages in a tree. Highlighted packages are those specified in the manifest.
+
+The package tree can also be inverted (`-i`), to see which packages require a specific dependencies.
+
+##### Arguments
+
+- `REGEX` optional regex of which direct dependencies to filter the tree to, or which dependencies to start with when inverting the tree.
+
+##### Options
+
+- `--invert (-i)`: Invert the dependency tree, that is given a `REGEX` pattern that matches some packages, show all the packages that depend on those.
+- `--platform <PLATFORM> (-p)`: The platform to list packages for. Defaults to the current platform
+- `--manifest-path <MANIFEST_PATH>`: The path to [manifest file](configuration.md), by default it searches for one in the parent directories.
+- `--environment`(`-e`): The environment's packages to list, if non is provided the default environment's packages will be listed.
+- `--frozen`: Install the environment as defined in the lockfile. Without checking the status of the lockfile. It can also be controlled by the `PIXI_FROZEN` environment variable (example: `PIXI_FROZEN=true`).
+- `--locked`: Only install if the `pixi.lock` is up-to-date with the [manifest file](configuration.md)[^1]. It can also be controlled by the `PIXI_LOCKED` environment variable (example: `PIXI_LOCKED=true`). Conflicts with `--frozen`.
+- `--no-install`: Don't install the environment for pypi solving, only update the lock-file if it can solve without installing. (Implied by `--frozen` and `--locked`)
+
+```shell
```

Comment:
```suggestion
```
This line is not needed.

Reply:
Helps if I look at how things are being rendered by in the `list` docs, rather than blindly copying them. I've removed both of them.

Reply:
Thanks!